course/resources/blurb-rqm.md

```diff
@@ -0,0 +1,3 @@
+We are an interdisciplinary group of graduate students interested in launching a third or fourth year undergraduate course in Reproducible Quantitative Methods (RQM). The curriculum is based on an [open curriculum](https://cbahlai.github.io/rqm-template/) developed by Dr. Christie Bahlai at Michigan State University through a Mozilla Science Lab fellowship. It is specifically designed to meet a growing demand for reproducible, openly accessible, statistically thorough, and well documented science. Although daunting at times, this approach to research presents an opportunity to improve the scientific practice, encourage broader participation in the scientific enterprise, and facilitate meeting the increasingly strict requirements from funding agencies in terms of data accessibility and reproducibility.
+
+The RQM course takes an approach that both produces conventional research products and trains students to work in accordance to sound practices within the framework laid out above. We will be teaching a pilot version of this course as a seminar topic in the department of Ecology and Evolutionary Biology (EEB) during the upcoming fall semester. If the pilot offering is positively met, we aim to offer a full version of the course the upcoming academic year, preferably as a cross-departmental effort between EEB and the department of Computer Science, and we are looking for a professor interested in championing this initiative.
```

Comment:
1. Add the date to the fall semester: "upcoming fall 2017 semester". 
2. Maybe remove "if the pilot offering is positively met", since we want to assume it will be :smile:
3. Change "the upcoming academic year" to "during the 2018-2019 academic year"
4. Possibly capitalize both instances of "department". 
5. "Endorsing" vs. "sponsoring" vs. "championing" - my vote would be for "supporting this initiative in order to create a new course offering" or something like that. I agree that champion might be a bit too strong, since we really want to be doing all of the work of the instructor. 
6. Do we need to add something about who to contact? Or let Francois add that part?